pkg/manager/manager.go

```diff
@@ -490,6 +501,14 @@ func (o Options) AndFrom(loader config.ControllerManagerConfiguration) (Options,
 		o.CertDir = newObj.Webhook.CertDir
 	}
 
```

Comment:
> pkg/manager/manager.go:453:1: cyclomatic complexity 31 of func `(Options).AndFrom` is high (> 30) (gocyclo)

Should I split up the func? Not sure if there's a good alternative to splitting up